test/integration/tests/buybacks.ts

```diff
@@ -78,6 +85,66 @@ describe('e2e-buybacks', function () {
     });
   });
 
+  describe('LUSD LBP', async function () {
+    it('mints appropriate amount and swaps', async function () {
+      const feiLusdLBPSwapper: BalancerLBPSwapper = contracts.feiLusdLBPSwapper as BalancerLBPSwapper;
+      const feiLusdLBP: IWeightedPool = contracts.feiLusdLBP as IWeightedPool;
+      const balancerVault: IVault = contracts.balancerVault as IVault;
+
+      const LUSD_HOLDING_ADDRESS = '0x66017D22b0f8556afDd19FC67041899Eb65a21bb';
+      await forceEth(LUSD_HOLDING_ADDRESS);
+      const lusdSigner = await getImpersonatedSigner(LUSD_HOLDING_ADDRESS);
+
+      await contracts.lusd.connect(lusdSigner).approve(balancerVault.address, ethers.constants.MaxUint256);
+
+      await balancerVault.connect(lusdSigner).swap(
+        {
+          poolId: await feiLusdLBP.getPoolId(),
+          kind: 0, // given in
+          assetIn: contracts.lusd.address,
+          assetOut: contracts.fei.address,
+          amount: ethers.constants.WeiPerEther.mul(300_000),
+          userData: '0x'
+        },
+        {
+          sender: LUSD_HOLDING_ADDRESS,
+          fromInternalBalance: false,
+          recipient: LUSD_HOLDING_ADDRESS,
+          toInternalBalance: false
+        },
+        ethers.constants.WeiPerEther.mul(200_000),
+        ethers.constants.WeiPerEther // huge deadline
+      );
```

Comment:
would be good to check balances in between each swap. doesn't have to be exact, just asset that the balance of the swapped-in asset increases in the bool and the balance of the swapped-out asset decreases.

Reply:
added test